client/options.go

```diff
@@ -175,11 +174,9 @@ func WithGuaranteeTimestamp(gt uint64) SearchQueryOptionFunc {
 	}
 }
 
-// WithTravelTimestamp specifies time travel timestamp
+// Deprecated: time travel is not supported since v2.3.0
 func WithTravelTimestamp(tt uint64) SearchQueryOptionFunc {
-	return func(option *SearchQueryOption) {
-		option.TravelTimestamp = tt
-	}
+	return nil
```

Comment:
nil may cause panicking when invoking return option. 
```suggestion
	return func(option *SearchQueryOption){}
```